runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/state/StateRequestHandlers.java

```diff
@@ -178,10 +179,10 @@ static BagUserStateHandlerFactory unsupported() {
    * {@link MultimapSideInputHandlerFactory} are cached.
    */
   public static StateRequestHandler forMultimapSideInputHandlerFactory(
-      ExecutableProcessBundleDescriptor processBundleDescriptor,
```

Comment:
Why did you want to break this chain by not passing in the ExecutableProcessBundleDescriptor?

Also, did you meant to make this change within this PR?

Reply:
ExecutableProcessBundleDescriptor contains API descriptors for the state and data services. We do not have access to those in the places where we construct state handlers in the Flink runner and in general those should not be required when defining handler behavior.